maven build failure

```
[INFO] Scanning for projects...
[INFO]
[INFO] -------------------------< au.csiro:fhir-owl >--------------------------
[INFO] Building FHIR OWL 1.1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ fhir-owl ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] Copying 1 resource
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ fhir-owl ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 9 source files to ./fhir-owl/target/classes
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] ./fhir-owl/src/main/java/au/csiro/fhir/owl/FhirOwlService.java:[32,24] cannot find symbol
  symbol:   class PostConstruct
  location: package javax.annotation
[ERROR] ./fhir-owl/src/main/java/au/csiro/fhir/owl/FhirOwlService.java:[101,4] cannot find symbol
  symbol:   class PostConstruct
  location: class au.csiro.fhir.owl.FhirOwlService
[INFO] 2 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.289 s
[INFO] Finished at: 2020-03-03T14:49:16-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project fhir-owl: Compilation failure: Compilation failure:
[ERROR] ./fhir-owl/src/main/java/au/csiro/fhir/owl/FhirOwlService.java:[32,24] cannot find symbol
[ERROR]   symbol:   class PostConstruct
[ERROR]   location: package javax.annotation
[ERROR] ./fhir-owl/src/main/java/au/csiro/fhir/owl/FhirOwlService.java:[101,4] cannot find symbol
[ERROR]   symbol:   class PostConstruct
[ERROR]   location: class au.csiro.fhir.owl.FhirOwlService
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

--- a/pom.xml
+++ b/pom.xml
@@ -79,6 +79,11 @@
             <artifactId>commons-cli</artifactId>
             <version>1.4</version>
         </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.3.2</version>
+        </dependency>
     </dependencies>
 
     <build>

